docs/RUNNING-LOCALLY.md

```diff
@@ -37,3 +37,46 @@ cd ../
 > development environment testing. This file should be based on the
 > `appsettings.json` file on EdFi.DataManagementService.Api.Tests.Integration
 > folder.
+## Running Unit Tests and Generate Code Coverage Report
+To run the unit tests locally, execute the following command in a PowerShell
```

Comment:
Please add an extra line above and below the header (line 40). Headers should always have space around them.

Reply:
Fixed